packages/components/src/__tests__/ConformityCredential.test.tsx

```diff
@@ -0,0 +1,281 @@
+      const url = 'https://example.com/credential';
+      (getJsonDataFromConformityAPI as jest.Mock).mockResolvedValue(url);
+      (getCredentialByPath as jest.Mock).mockReturnValue(url);
+
+      await act(async () => {
+        render(
+          <ConformityCredential
+            credentialRequestConfigs={credentialRequestConfigs}
+            storedCredentials={storedCredentials}
+          />,
+        );
+      });
+
+      await act(async () => {
+        const credentialButton = screen.getByText('Deforestation Free Assessment');
+        fireEvent.click(credentialButton);
+      });
+
+      expect(screen.findByText(credentialRequestConfigs[0].credentialName)).not.toBeNull();
+      expect(screen.findByText(url)).not.toBeNull();
+      expect(document.querySelector('table')).not.toBeNull();
+    });
+
+    it('should save credential as object when trigger onClickStorageCredential function', async () => {
+      const apiResp = {
+        credentialSubject: {
+          name: 'Jane Smith',
+          degree: {
+            type: 'BachelorDegree',
+            name: 'Bachelor of Science and Arts',
+            degreeSchool: 'Example University',
+          },
+          id: 'did:example:1234',
+        },
+        issuer: {
+          id: 'did:example:1234',
+        },
+        type: ['VerifiableCredential', 'UniversityDegreeCredential'],
+        proof: {
+          type: 'JwtProof2020',
+          jwt: 'example-jwt',
+        },
+      };
+
+      (getJsonDataFromConformityAPI as jest.Mock).mockResolvedValue(apiResp);
+      (getCredentialByPath as jest.Mock).mockReturnValue(apiResp);
+      (checkStoredCredentials as jest.Mock).mockReturnValue({ ok: true, value: '' });
+      (uploadJson as jest.Mock).mockResolvedValue('https://storage.example.com/credential');
+
+      await act(async () => {
+        render(
+          <ConformityCredential
+            credentialRequestConfigs={credentialRequestConfigs}
+            storedCredentials={storedCredentials}
+          />,
+        );
+      });
+
+      await act(async () => {
+        const credentialButton = await screen.findAllByText('Deforestation Free Assessment');
+        fireEvent.click(credentialButton[0]);
+      });
+
+      expect(screen.findAllByLabelText(credentialRequestConfigs[0].credentialName)).not.toBeNull();
+      expect(screen.findByText('https://storage.example.com/credential')).not.toBeNull();
+      expect(document.querySelector('table')).not.toBeNull();
+    });
+
+    // error case
+    it('should throw error when url in credentialRequestConfig props not valid after triggering onClickStorageCredential function', async () => {
+      const newCredentialRequestConfigs = [
+        {
+          url: '',
+          params: {},
+          options: {
+            headers: [],
+            method: 'POST',
+          } as FetchOptions,
+          credentialName: 'Deforestation Free Assessment',
+          credentialPath: '',
+          credentialAppExclusive: '',
```

Comment:
the implementation of `credentialAppExclusive` looks like opposite meaning with the name